### ISSUE TITLE
Emoji picker seems to close, but reappears

### ISSUE THREAD

**Issue by mikedilger** [mikedilger/gossip#830]
When you pick an emoji, the picker disappears.  But then it reappears when you scroll or go elsewhere and you have to click outside of it to make it go away again.

**Comment by mikedilger** [comment]
See #866 that was closed as a dup it has more details.

**Comment by ioseph-silva** [comment]
While there is no other solution, I think we can add this after line 973 to make it close when scrolling:

``` rust
if app.is_scrolling() {
	ui.close_menu();
 }
```

add after line 973 in this file:
https://github.com/mikedilger/gossip/blob/c7e92933eba8569905e0f4503853b9349a5039b2/gossip-bin/src/ui/feed/note/mod.rs#L964-L973

**Comment by ioseph-silva** [comment]
Yes, the emoji picker closes when I scroll the feed. The mouse needs to be outside the picker to allow scrolling.

I think I should have linked the unstable branch above. Nevertheless, I committed it here: 1b88b4b6ae3a1d37c670766c2af1020bf62b83c4

**Comment by mikedilger** [comment]
on unstable.
